src/components/inputs/TextInput.tsx

```diff
@@ -21,6 +21,7 @@ interface TextInputProps extends React.InputHTMLAttributes<HTMLInputElement> {
   inputGroupContent?: JSX.Element;
   submitButtonText?: string | React.ReactNode;
   buttonDisabled?: boolean;
+  isClearBtnVisible?: boolean;
```

Comment:
Why was this added?

Reply:
That was a temporary variable i used before creating the functions to pass to 'clearInputAction'. I've removed it now.